src/Message/VoidResponse.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace MyOnlineStore\Omnipay\KlarnaCheckout\Message;
+
+use Omnipay\Common\Message\ResponseInterface;
+
+final class VoidResponse extends AbstractResponse implements ResponseInterface
+{
+    /**
+     * @inheritDoc
+     */
+    public function isSuccessful()
+    {
+        return true;
```

Comment:
Perhaps an API response based check could be added here as well to determine if the void operation was successful.